lucene/core/src/java/org/apache/lucene/document/LatLonShape.java

```diff
@@ -341,6 +341,16 @@ public static Query newGeometryQuery(
     }
   }
 
+  /**
+   * Factory method for creating the {@link LatLonShapeDocValues}
+   *
+   * @param bytesRef {@link BytesRef}
+   * @return {@link LatLonShapeDocValues}
+   */
+  public static LatLonShapeDocValues buildLatLonShapeDocValues(BytesRef bytesRef) {
```

Comment:
Let's go ahead and make this consistent w/ all of the other factory methods.

```suggestion
  public static LatLonShapeDocValues createLatLonShapeDocValues(BytesRef bytesRef) {
```